docs/source/kale.loaddata.rst

```diff
@@ -46,6 +46,14 @@ kale.loaddata.multi\_domain module
    :undoc-members:
    :show-inheritance:
 
+kale.loaddata.multiomics\_gnn\_dataset module
```

Comment:
This name needs. GNN is a model so use it in the name of dataset to load is inappropriate. Please examine other APIs under loaddata. Also think about whether it should be singular or plural.

Reply:
test comment